packages/Tabs/index.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div>
+    <div ref="tabs" :class="$style['tabs']">
+      <div
+        v-for="(name, index) in names"
+        :key="`tab${index}`"
+        :class="[$style['tab'], { [$style['active']]: name === currentTab }]"
+        @click="e => clickTab(e.target, name)"
+      >
+        {{ name }}
+      </div>
+    </div>
+    <slot :name="activeTabContent"></slot>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Tabs',
+  props: {
+    /**
+     * tab names
+     */
+    names: {
+      type: Array,
+      default: () => []
+    }
+  },
+  data() {
+    return {
+      currentTab: this.names.find((_, i) => i === 0)
+    }
+  },
+  computed: {
+    tabs() {
```

Comment:
Making `this.$refs.tabs` short may cause ambiguity.
`this.tabs` may looks like it's an array and does not seems to be a DOM.